pkg/k8sapi/discovery.go

```diff
@@ -256,7 +258,7 @@ func (d *Controller) handlePodUpdate(key string) error {
 		d.workerPodsLock.Lock()
 		defer d.workerPodsLock.Unlock()
 
-		log.Tracef("Update for pod %s: %+v, %+v", podName, pod.Status, pod.Spec)
+		log.Infof("Update for pod %s: %+v, %+v", podName, pod.Status, pod.Spec)
```

Comment:
This will increase the amount of logging a bit, but not as much as before, since we have limited the watcher quite a lot.